### ISSUE TITLE
Cannot debug web workers in Firefox

### ISSUE THREAD

**Issue by mike-lischke** [firefox-devtools/vscode-firefox-debug#248]
I'm coming from a bug opened against the [vscode js debugger](https://github.com/microsoft/vscode-js-debug/issues/932#event-4414596169) and had to learn that Firefox is actually the culprit, which makes the trouble when trying to debug web worker code. My vscode launch config is.

```json
        {
            "type": "firefox",
            "request": "launch",
            "name": "Launch MSG on FF",
            "url": "https://localhost:3001",
            "webRoot": "${workspaceFolder}",
            "clearConsoleOnReload": true,
            "preLaunchTask": "tsc: watch",
            "keepProfileChanges": false,
            "reAttach": true,
            "profile": "dev",
            "enableCRAWorkaround": true,
            "firefoxArgs": [
                "-devtools",
            ]
        },
```

Normal debugging works pretty well, even though breakpoints are often shifted when a hot code replace happened. However, setting a breakpoint in web worker code in vscode does not even validate the breakpoint, let alone stop the debugger there. I would hate to switch to Chrome to allow debugging such code.

**Comment by mike-lischke** [comment]
@hbenl Sorry to ping you directly, but the issue tracker didn't get much attention in the past few months. And meanwhile a few pretty serious problems appeared with the FF debug tools. One is this problem described here. Another the missing debug ouput issue #247, from which I also suffer. I would be very happy if you could take a look at these problems or at least provide some feedback here. Many thanks.

**Comment by hbenl** [comment]
@mike-lischke Yeah, I've been very busy recently and didn't have time to keep this extension up to date with the latest changes in Firefox' debug protocol. But I'm working on it now and have fixes for #247 and this issue almost ready.

**Comment by hbenl** [comment]
Fixed in version 2.9.3
